packages/authentication/test/unit/fixtures/mock-strategy.ts

```diff
@@ -4,7 +4,7 @@
 // License text available at https://opensource.org/licenses/MIT
 
 import {Strategy, AuthenticateOptions} from 'passport';
-import {PassportRequest} from '../../..';
+import {Request} from 'express';
```

Comment:
Shouldn't this be imported from `@loopback/rest`?

Reply:
Here is my reasoning: `MockStrategy` is extending from Passport `Strategy`, which uses `Request` from `express`.

If we ever change `@loopback/rest` to export `Request` as a different type, then  MockStrategy still needs to keep using `Request` from `express`, because that's what the `Strategy` interface requires.